.pylintrc

```diff
@@ -0,0 +1,575 @@
+# Move to pyproject.toml file once pylint 2.5 is released
```

Comment:
Might be easier to just use [flake8](https://github.com/tophat/syrupy/compare/add-flake8), pylint has a lot of configuration needed